src/js/module.js

```diff
@@ -126,18 +126,21 @@ iotjs_module_t.resolveFilepath = function(id, directories) {
 };
 
 
-iotjs_module_t._resolveFilepath = function(id, root) {
+iotjs_module_t._resolveFilepath = function(id, root, ext_index) {
```

Comment:
Where is the `ext_index` used?

Reply:
It used only in require.resolve for search a right ext from [js, json, node].

Reply:
And how can I do this bench, in my plan, I will try `require` function many times by before add `resolve` method and after, then try `require.resolve`.

Reply:
You should clean up the cache, either :)

Reply:
Yeah!